src/ppu.jakt

```diff
@@ -421,7 +421,13 @@ class PPU {
         }
         for sprite in (0u8..63u8).inclusive() {
             let sprite_addr = sprite * 4u8
-            let sprite_y = .oam[sprite_addr] + 1
+            mut sprite_y = 0u8
+            //FIXME: This only fixes an overflow crash, it may not be correct emulation
```

Comment:
Looks like the fix here, from the nesdev docs is:

"Hide a sprite by writing any values in $EF-$FF here" (from https://www.nesdev.org/wiki/PPU_OAM)

So if we see an `.oam[sprite_addr]` >= 0xef we should continue the loop and skip that sprite.